packages/opentelemetry-tracing/test/Tracer.test.ts

```diff
@@ -220,4 +221,37 @@ describe('Tracer', () => {
     assert.strictEqual(context.traceFlags, TraceFlags.NONE);
     span.end();
   });
+
+  it('should start an active span', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', span => {
+      try {
+        return 1
+      } finally {
+        span.end();
+      }
+    }), 1);
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', {}, span => {
+      try {
+        return 1
+      } finally {
+        span.end();
+      }
+    }), 1);
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', {}, context.active(), span => {
```

Comment:
Please assert that the context sent to the method is used

```typescript
ctx = context.set(someKey, 'somevalue');

tracer.startActiveSpan('my-span', {}, context.active(), span => {
	assert.strictEqual(context.get(someKey), 'somevalue')
});
```

Reply:
done